lbr_demos/lbr_demos_py/lbr_demos_py/move_2_cart.py

```diff
@@ -0,0 +1,208 @@
+#! /usr/bin/env python3
+
+import rclpy
```

Comment:
you'll have to run the `black` formatter to pass the linting tests for all python files